api_mt/index.md

```diff
@@ -0,0 +1,137 @@
+# __HOW TO CONNECT POPULAR MTs TO CAT TOOLS__
+
+> ___The biggest challenge is to combine technology and the human side without losing the latter___ — Gordana Antonijević, Head of Translation Unit, Veris
```

Comment:
What if she changes jobs?  :-D

I'd probably use fancy quotes here.

Em dash is good.

Reply:
I removed the: ", Head of Translation Unit, Veris" part. I order to put smart quotes I used htms (afaik there is a problem with them in markdown) the line looks like this now:  &ldquo;The biggest challenge is to combine technology and the human side without losing the latter&rdquo; Visual Studio Code shows in the preview that it works. Do let me know if this way might display kinda "funky" on the website. Thank you.

Reply:
There should be no problem with smart quotes in Markdown:

> “test”